gravitee-apim-console-webui/src/management/api/list/api-list.component.ts

```diff
@@ -187,6 +256,52 @@ export class ApiListComponent implements OnInit, OnDestroy {
         size: initialPageSize,
       },
     };
+    const queryParams = this.activatedRoute.snapshot.queryParams;
+    if (queryParams.apiTypes) {
+      // If only one value is present in the query params it is considered as a string
+      if (typeof queryParams.apiTypes === 'string') {
+        this.filters.apiTypes = [queryParams.apiTypes];
+      } else {
+        this.filters.apiTypes = queryParams.apiTypes;
+      }
```

Comment:
normally you ca replace this by a lodash `castArray` 
like : `this.filters.apiTypes = castArray(queryParams.apiTypes)`